src/types/custom_metadata_types.ts

```diff
@@ -91,12 +92,16 @@ export function isCustomMetaData(tags: unknown): tags is CustomMetaData {
 	}
 
 	for (const [key, metaData] of Object.entries(tags)) {
+		// consider using a sqwitch..case
```

Comment:
```suggestion
		switch (key) {
			case 'metaDataJson':
			case 'metaDataGqlTags':
			case 'dataGqlTags':
				if (!isCustomMetaDataJsonFields(metaData)) {
					return false;
				}
				break;
			default:
				break;
		}
```
I think it's basically the same number of lines but much clearer.

Reply:
c771e0b925116bf501f6db0b6d97b25ce3fe4ede

Reply:
Nit: I didn't add a default there, should I?

Reply:
I did: 8ae049ea8e1e4c6318453e3c495a66c014a693f7